BibBuddy/app/src/main/java/de/bibbuddy/BookFormFragment.java

```diff
@@ -64,7 +64,9 @@ public void handleOnBackPressed() {
           authorList.addAll(model.getAuthorList(bookId));
         }
 
+        ((MainActivity) getActivity()).setVisibleImportShareButton(View.INVISIBLE, View.INVISIBLE);
```

Comment:
I would set the buttons to invisible by default in the xml and then show them when needed.

Reply:
Do you mean that if the buttons in header.xml are invisible, then I do not need this setVisibleImportShareButton()-method here?

Reply:
Exactly.

Reply:
This suggestion did not work for me. 
The visibility of the buttons is not changed from Fragment to Fragment when I set the buttons as invisible as default and show them only in the LibraryFragment, BookFragment, and BookNotesView. 
Reason: the buttons are shown in LibraryFragment, then if you go for example to HomeFragment -> the buttons are also shown in HomeFragment, just because the visibility state is not changed in the HomeFragment.